src/components/ApolloResults.vue

```diff
@@ -19,7 +19,7 @@ import { SearchResultItemConnection } from "@octokit/graphql-schema";
 
 import Results from "./Results.vue";
 import Issue from "./Issue.vue";
-import SearchQuery from "../graphql/Search.gql";
+import SearchQuery from "../graphql/Search.js";
```

Comment:
It didn't like the `.gql` extension - I'm guessing that's something one of the deprecated dependencies I had to remove provided.